test/Microsoft.Azure.WebJobs.Host.EndToEndTests/SingletonEndToEndTests.cs

```diff
@@ -411,7 +411,7 @@ public async Task SingletonJob_StorageAccountOverride()
         public class TestJobs
         {
             private const string Secondary = "SecondaryStorage";
-            public const string LeaseBlobRootPath = "Microsoft.Azure.WebJobs.Host.EndToEndTests.SingletonEndToEndTests+TestJobs";
+            public const string LeaseBlobRootPath = "Microsoft.Azure.WebJobs.Host.EndToEndTests.SingletonEndToEndTestsV2+TestJobs";
```

Comment:
Where is this SingletonEndToEndTestsV2 naming coming from? I wouldn't think we'd need the V2 suffix.